Eplant/views/ChromosomeViewer/Viewer/GeneList.tsx

```diff
@@ -0,0 +1,247 @@
+// -------
+// IMPORTS
+// -------
+import React, { FC, useState } from "react";
+import Draggable from "react-draggable";
+
+import GeneticElement from "@eplant/GeneticElement";
+import arabidopsis from "@eplant/Species/arabidopsis";
+import {
+	useActiveGeneId,
+	useCollections,
+	useGeneticElements,
+	useSetActiveGeneId,
+	useSetCollections,
+	useSetGeneticElements,
+	useSidebarState
+} from "@eplant/state"
+import CloseIcon from '@mui/icons-material/Close';
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import DialogTitle from "@mui/material/DialogTitle";
+import IconButton from "@mui/material/IconButton";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from "@mui/material/ListItemText";
+import Popover from "@mui/material/Popover";
+import useTheme from "@mui/material/styles/useTheme";
+import Typography from "@mui/material/Typography";
+
+import { GeneIcon } from "../icons";
+import { GeneArray, GeneItem } from "../types";
+
+// TYPES
+interface GeneListProps {
+	id: string,
+	start: number,
+	end: number,
+	anchorOrigin: Array<number>
+}
+
+
+//----------
+// COMPONENT
+//----------
+const GeneList: FC<GeneListProps> = ({
+	id,
+	start,
+	end,
+	anchorOrigin
+}) => {
+
+
+	const [loading, setLoading] = useState<boolean>(true);
+	// gene list
+	const [geneList, setGeneList] = useState<GeneArray>([{
+		id: "",
+		start: 0,
+		end: 0,
+		strand: "",
+		aliases: [],
+		annotation: ""
+	}]);
+	const [selectedIndex, setSelectedIndex] = useState<number | null>(null);
+	const [selectedGene, setSelectedGene] = useState<GeneItem | null>(null)
+	// gene info popup
+	const [open, setOpen] = useState(false);
+
+	// Other/Global State
+	const [activeGeneId, setActiveGeneId] = useActiveGeneId()
+	const geneticElements = useGeneticElements()
+	const setGeneticElements = useSetGeneticElements()
+	const collections = useCollections()
+	const setCollections = useSetCollections()
+	const theme = useTheme()
+
+
+
+	//------------------
+	// Helper Functions
+	//------------------
+	React.useEffect(() => {
```

Comment:
```suggestion
	useEffect(() => {
```
Import individual exported members from react whenever possible